homedir/homedir.go

```diff
@@ -1,3 +1,5 @@
+// https://github.com/mitchellh/go-homedir
```

Comment:
this file is just copied from `go-homedir`. probably a good idea to use the official copy, at some point.